packages/jsii-pacmak/lib/targets/go.ts

```diff
@@ -2,11 +2,12 @@ import { CodeMaker } from 'codemaker';
 import { Documentation } from './go/documentation';
 import { Assembly } from 'jsii-reflect';
 import { Rosetta } from 'jsii-rosetta';
-import { join } from 'path';
 import { RootPackage } from './go/package';
 import { IGenerator } from '../generator';
 import { Target, TargetOptions } from '../target';
 import { goPackageName } from './go/util';
+import * as fs from 'fs-extra';
+import * as path from 'path';
```

Comment:
May I be so bold as to ask we sort those imports (external / not-starting with `.` first, then internal / starting with `.`)?